### PR TITLE
Photoshop cheatsheet : Alias Added

### PR DESCRIPTION
**Pull request by ankushg07**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [ ] New Instant Answer
  - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [x] Enhancement: **`Photoshop cheatsheete}: {Alias Added}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

What changes does this PR introduce? 
This will allow user to trigger the query using alias 
###### People to notify

@therealkbhat

---

Instant Answer Page: https://duck.co/ia/view/photoshop_cheat_sheet

[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @therealkbhat

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ankushg07 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by therealkbhat** [comment]
@ankushg07 The changes look fine to me, but as mentioned above, we're prioritising PRs related to the Programming Mission right now.

**Comment by ankushg07** [comment]
@therealkbhat Programming Mission? Could you please explain what this all about and what kind of PR will be on priority

File: share/goodie/cheat_sheets/json/photoshop.json [hunk 1]
```diff
@@ -5,6 +5,11 @@
     "metadata": {
         "sourceName": "PhotoshopCheatSheet"
     },
+    "aliases": [
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/photoshop.json]: you need to make all the aliases lower case

File: share/goodie/cheat_sheets/json/photoshop.json [hunk 2]
```diff
@@ -5,6 +5,11 @@
...
+        "Adobe Photoshop",
+        "Photoshop CS6",
```
Comment (gautamkrishnar) [inline share/goodie/cheat_sheets/json/photoshop.json]: is this cheatsheet specific to CS6 version?

**Comment by gautamkrishnar** [comment]
@ankushg07 are you still up to this?. Just pinging...

**Comment by ankushg07** [comment]
@gautamkrishnar  yeah sorry for not replying 
was busy with end semester exam 
No i dont think its completely for cs6 but creator of this cheat can clarify this

**Comment by ankushg07** [comment]
@gautamkrishnar but i think cs6 is most commonly used version so it should worked on it too

**Comment by pjhampton** [comment]
Closing as this PR looks to have gone stale.

If you are looking for something more substantial to work through that will have a big impact for the users of DuckDuckGo, please check out the [DuckDuckHack Issues Table](https://duckduckhack.com/issues). 🚀  😄

**Pull request closed without merging**